dispatcher/nm-dispatcher.c

```diff
@@ -620,14 +666,13 @@ find_scripts (const char *str_action)
 			g_warning ("find-scripts: Cannot execute '%s': %s", path, err_msg);
```

Comment:
I think we should also skip files with size of 0. We allow such empty files to shadow another file, but it shouldn't actually be run.

Reply:
Systemd shadows things with /dev/null symlinks. I think that we should do as well, but without a warning.